structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,125 @@
+
+Community Collab Managers are the same as Community Managers of the Quilt Community Team.
+
+### Collaborators
+
+Collaborators are moderators of one or more communities in the program. They have access to the server to the moderation part of the server, such as general moderator chats and case threads. Access is also given to the event category.
+
+### Event Planners
+
+Event Planners are members of the staff team of one or more communities in the program, who focuses on event organisation. They have access to the event part of the server.
+
+### Project Roles
+
+Other roles may be given surrounding the different projects happening inside the program. Those roles are informal and will depend on each project.
+
+### Invitees
+
+Invitees are temporary users, with no access to the rest of the server. They only have access to temporary channels required by the process they are visiting for.
+
+## Voting process
+
+If required, a vote among the Community Collab group can be called by a Manager. A poll and an associated thread are created in an internal channel designated for that purpose, including as many details as possible.
+
+Each community gets a single vote representing its member. They are free to write their own process for determining their vote, but it must follow those rules:
+
+- This process must be democratic and give a voice to any community member who is able to join the Community Collab server under the Collaborator role.
+- Each eligible voter must have single, equal voting power.
+- The process can remove the voting power under some inactivity threshold.
+
+Once a community has decided their vote, one member can report their vote in the thread, mentioning the community name, and the option being voted for, with corresponding proof (such as a screenshot of an internal poll).
+
+Each community also has the option of skipping its vote if none of its members has an opinion on the poll. Such instances should be reported the same way.
+
+Polls must include a closing date of at least ten days. If required, the closing date can be pushed away by a maximum of a month, upon the informal agreement with the majority of the other communities. During that period, any Collaborator can use their veto right (see below).
+
+Once the poll is closed, the vote can be interpreted, depending on the specific process this vote is part of, if any.
+
+## Veto rights
+
+While the vote is active, any Collaborator can use their veto right by sending a message inside the thread, detailing their reasoning.
+
+If another Collaborator believes the reasoning isn’t satisfying, they can start the overruling process. If so, another poll is set up, if more than half of the individual votes are positive after three days, the veto is discarded.
+
+If no Collaborator overrules the veto after three days, the vote is considered as failed right away. The Managers are free to only communicate the result to the Invitees at a later date to keep the veto secret.
```

Comment:
Why are we using the generic term for anyone with temporary access here? I think the definition of "Invitee" might need to be pulled in to better represent what youre envisioning, or declare a different term altogether